src/lib/format/tests/TestDecoding.cpp

```diff
@@ -274,7 +274,7 @@ void TestFullDataDecoding(nlTestSuite * inSuite, void * inContext)
                    "          endpoint_id: 0\n"
                    "          cluster_id: 49 == 'NetworkCommissioning'\n"
                    "          attribute_id: 65532 == 'featureMap'\n"
```

Comment:
`featureMap` is not the right name. `FeatureMap` is the attribute name

Reply:
This is from the matter file. I believe we `lowerCamelCase` attribute names. We can update it if needed, but that seems a separate concern.